trollmoves/server.py

```diff
@@ -692,21 +739,40 @@ def process_notify(orig_pathname, publisher, pattern, attrs):
         LOGGER.debug('We have a match: %s', orig_pathname)
 
     pathname = unpack(orig_pathname, **attrs)
+    if "request_port" in attrs:
+        msg = create_message_with_request_info(pathname, orig_pathname, attrs)
+    else:
+        msg = create_message_with_remote_fs_info(pathname, attrs)
```

Comment:
So if the `request_port` isn't defined, it means that there won't be a need for a matching Client to make transfer requests? And then the consumer (Trollflow2/Satpy) would read then listen to directly the messages sent by the Server process, and need to read the data directly from the original source (reception server or such) using `fsspec` and have the required credentials available at their end?

Reply:
yes, this is correct.